Source/CarthageKit/SwiftShims.swift

```diff
@@ -4,4 +4,12 @@ extension Sequence {
 		return try flatMap(transform)
 	}
 }
+
+func == (lhs: FrameworkType??, rhs: FrameworkType) -> Bool {
```

Comment:
This `??` is a code smell and I think we should avoid it. Instead of adding this `==` let's get rid of the `??` in the caller code.

Reply:
The problem is, that `BuildOptions.frameworkType` returns `Result<FrameworkType?, ...>`, so if you get the value of this result, it became `FameworkType??`. What should I do with this?